compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/test/tiling.mlir

```diff
@@ -1595,11 +1598,152 @@ module attributes { transform.with_named_sequence } {
 
 // -----
 
+func.func @attention_float_mask(%query: tensor<192x1024x64xf32>, %key: tensor<192x1024x64xf32>, %value: tensor<192x1024x64xf32>, %mask: tensor<192x1024x1024xf32>) -> tensor<192x1024x64xf32> {
```

Comment:
Personally, I would not add any tiling tests for iree_linalg_ext.attention op. We are moving to make this a frontend only operation, and should really just not add more stuff for it. I'm going to send a pr next week to remove tiling for this op. I'll let @raikonenfnu decide if we want to keep it in this pr.